rpc-interface/src/network.rs

```diff
@@ -15,4 +15,7 @@ pub trait NetworkInterface {
 
     /// Returns a list with the IDs of all our peers.
     async fn get_peer_list(&self) -> RPCResult<Vec<String>, (), Self::Error>;
+
+    /// Returns the version of the local client.
```

Comment:
To me it's more the remote, but we can make it clear by saying:
```suggestion
    /// Returns the version of the client the RPC server is running.
```